src/analysis/retail/monk/mistweaver/modules/spells/TearOfMorning.tsx

```diff
@@ -0,0 +1,78 @@
+import talents from 'common/TALENTS/monk';
+import spells from 'common/SPELLS/monk';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { HealEvent } from 'parser/core/Events';
+import { calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import Statistic from 'parser/ui/Statistic';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { SpellLink } from 'interface';
+import { formatNumber } from 'common/format';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+
+const TEAR_OF_MORNING_INCREASE = 0.2;
+class TearOfMorning extends Analyzer {
+  static dependencies = {};
+
+  duplicatedRems: number = 0;
```

Comment:
This analyzer seems unfinished since we don't account for duplicated Renewing mists, you made the variable but I dont think you implemented the logic

Reply:
Won't be in this PR